pkg/image/validation/kubernetes.go

```diff
@@ -253,6 +276,36 @@ func validateRepo(repo *image.HelmRepository, seenHelmRepos map[string]bool) []F
 	return failures
 }
 
+func validateHelmRepoArgs(repo *image.HelmRepository) []FailedValidation {
+	var failures []FailedValidation
+
+	if repo.SkipTLSVerify && repo.PlainHTTP {
+		failures = append(failures, FailedValidation{
+			UserMessage: fmt.Sprintf("Helm repository 'plainHTTP' and 'skipTLSVerify' fields for %q cannot both be true.", repo.Name),
+		})
+	}
+
+	if strings.HasPrefix(repo.URL, "http://") && !repo.PlainHTTP {
```

Comment:
This and the checks below seem quite complex. I'd generally suggest to `url.Parse()`, use a switch statement on the `Scheme` field. Feel free to ignore the suggestion if it is not applicable.

Reply:
I don't think a switch statement would work here but I did switch over to using `url.Parse()`.